application/controllers/handler.js

```diff
@@ -1444,7 +1444,7 @@ function enrichGroupMembers(group) {
     prev.push(curr.userid);
     return prev;
   }, []);
-  userids.push(group.creator.userid);
+  userids.push((group.creator.userid) ? group.creator.userid : group.creator);
```

Comment:
consider something like `if (typeof group.creator !== 'object') group.creator = {userid: group.creator}` if it makes sense in general